tests/queries/0_stateless/02483_test_reverse_dns_resolution.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Tags: no-parallel, no-fasttest
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+terra_ipv6="2600:1419:c400::214:c410"
+
+i=0 retries=300
+while [[ $i -lt $retries ]]; do
+    $CLICKHOUSE_CLIENT -q "select reverseDNSQuery('${terra_ipv6}')"
```

Comment:
Oh, that is true. The only reason this works is because I actually have that setting set. But in terms of DDoS won't it be the same case as the unit test?

Reply:
Yes, it will, that's why I suggest converting it to an integration test

Reply:
I just woke up, sitll a bit sleepy, sorry. I can move it to an integration tests, but there is a big chance it won't reproduce the crash anymore since DNS server is now local